### PR TITLE
DCM-54: Add user privileges to Configure DHIS Connection UI

### PR DESCRIPTION
**Pull request by Piumal1999**
## The issue I worked on
See [https://issues.openmrs.org/browse/DCM-54](https://issues.openmrs.org/browse/DCM-54) 

## Description of what I changed:
Updated the Configure DHIS Connection UI to behave according to the user privileges

## Related Thread and Preview:
https://talk.openmrs.org/t/dhis-connector-module-user-access-controlling/34059/10?u=piumal1999

### CODE REVIEW

**Comment by Piumal1999** [comment]
This PR is also Ready for review now
@jayasanka-sack @akshika47

File: omod/src/main/webapp/configureServer.jsp [hunk 1]
```diff
@@ -11,22 +13,26 @@
...
-    </tr>
-    <tr>
-      <td><spring:message code="dhisconnector.pass"/></td>
-      <td><input name="pass" placeholder="<hidden>" type="password" size="20"/></td>
-    </tr>
-    <tr>
-      <td/>
-      <td><input name="submit" type="submit" value="<spring:message code="dhisconnector.save" />"/></td>
-    </tr>
+    <openmrs:hasPrivilege privilege="View Connection,Manage Connection">
```
Comment (HerbertYiga) [inline omod/src/main/webapp/configureServer.jsp:16]: thanks @Piumal1999  why not pu the privilege on the entire form?

Comment (Piumal1999) [reply omod/src/main/webapp/configureServer.jsp:16]: I'll update it. Thanks for the suggestion.

Comment (Piumal1999) [reply omod/src/main/webapp/configureServer.jsp]: fixed with 53c2484

**Review by jayasanka-sack — APPROVED** [review]
Thanks for getting this done, @Piumal1999 !

**Pull request merged**